.tekton/coo-fbc-v4-14-pull-request.yaml

```diff
@@ -176,10 +195,13 @@ spec:
         - "true"
       workspaces:
       - name: output
```

Comment:
note the wrong change here, please do keep basic-auth, you should delete just the output workspace

Reply:
done